invenio_app_rdm/records_ui/views/filters.py

```diff
@@ -182,6 +183,25 @@ def custom_fields_search(field, field_value):
     )
 
 
+def custom_vocab_fields_search(field, field_value, sub_field):
```

Comment:
do we need a split from custom_fields_search? could we instead add a flag if is it a vocab and use a function above?